apps/web/components/booking/BookingListItem.tsx

```diff
@@ -1205,4 +1246,18 @@ const DisplayAttendees = ({
   );
 };
 
+const AssignmentReasonTooltip = ({ assignmentReason }: { assignmentReason: AssignmentReason }) => {
+  const { t } = useLocale();
+  const badgeTitle = assignmentReasonBadgeTitleMap(assignmentReason.reasonEnum);
+  return (
+    <Tooltip content={<p>{assignmentReason.reasonString}</p>}>
+      <div className="-mt-1">
```

Comment:
This was an interesting find. The `Tooltip` wouldn't work with the badge because what's returned is an element created with `React.createElement`. If I wrap it in a `<div>` it worked as expected.